lib/eventq/queue_worker.rb

```diff
@@ -149,7 +149,13 @@ def process_message(block, message, retry_attempts, acceptance_args)
       rescue => e
         EventQ.logger.error do
           "[#{self.class}] - Unhandled error while attempting to process a queue message. - #{e.message}" \
-          "#{e.backtrace.join("\n")}"
+          " - message.id: #{message.id}" \
+          " - message.created: #{message.created}" \
+          " - message.type: #{message.type}" \
+          " - message.content: #{message.content}" \
```

Comment:
I think we should exclude `content` from this to avoid any sensitive data being written into the logs.

Reply:
@vaughanbrittonsage content is mainly what we were aiming to see.
I asked @chrisbarber86 a while back about adding sensible data to logs when we were working on user authentication and I believe he said that logs should be seen as safe ground.
What do you think @chrisbarber86 ?

Reply:
A use case for this is the current GDPR problem. Since the data was manually deleted we now don't have a way to understand what the data was when it blew up.

Reply:
@chrisbarber86  can you please advise on this?

Reply:
I'd agree that we shouldn't include `content` - to be honest the primary thing we wanted to understand here was the line that it was failing on so we could better follow what was causing it.

`content` would have been nice but not essential.

Reply:
Agreed. I think we can close this PR then and leave just the stacktrace has we have.